CommunityToolkit.App.Shared/Controls/TitleBar/NativeMethods.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace CommunityToolkit.App.Shared.Controls;
 
+#pragma warning disable CA1060 // Move pinvokes to native methods class
```

Comment:
Not thrilled about just ignoring this for now, but it is a PITA to deal with otherwise, so for now it is the best solution. We should just file an issue about having CS/Win32 enabled for the sample app (if needed), this may also just go away once we ship https://github.com/CommunityToolkit/Labs-Windows/pull/459 in labs and switch to using the encapsulated control itself.

Reply:
Sure, done - https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/115